src/app/templates/app/license_requests.html

```diff
@@ -31,7 +31,9 @@
             <td>{{elem.submissionDatetime}}</td>
             <td>
               {% csrf_token %}
+              {% if github_login and authorized %}
               <button id='archive_button{{elem.id}}' class='btn btn-success' width='25px' value='Archive'>Archive</button>
+              {% endif %}
```

Comment:
IMO, the "Archive" table header should also be removed if user can't see the archive button. Same with Unarchive.
<img width="1110" alt="Screenshot 2020-04-30 at 12 18 35 PM" src="https://user-images.githubusercontent.com/17159160/80680648-ce7da380-8adc-11ea-9a0f-d1d53c912bfa.png">